### PR TITLE
fix(charts): Add lib and lib image values for pod-delete chart

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <uditgaurav@gmail.com>
- Add lib image values for the pod-delete experiment chart. This change was a part of litmus 1.4.1.

### CODE REVIEW

File: charts/kubernetes-chaos/values.yaml [hunk 1]
```diff
@@ -38,3 +38,7 @@ image:
   litmusStressNG:
     repository: litmuschaos/stress-ng
     tag: latest 
+
+  podDelete:
```
Comment (ksatchit) [inline charts/kubernetes-chaos/values.yaml:42]: can this be: litmusPodDeleteHelper

File: charts/kubernetes-chaos/templates/pod-delete.yaml [hunk 2]
```diff
@@ -72,6 +72,10 @@ spec:
       value: '5'
 
     - name: LIB
-      value: ''    
+      value: 'litmus'
+
+    - name: LIB_IMAGE
+      value: "{{ .Values.image.podDelete.repository }}:{{ .Values.image.podDelete.tag }}"
```
Comment (ksatchit) [inline charts/kubernetes-chaos/templates/pod-delete.yaml:78]: refer comment on values file

**Comment by ispeakc0de** [comment]
@uditgaurav Can you check the other changes as well.. to sync this helm charts w/ chaos-chart repo..  for example we don't have these changes in helm but needed for kind cluster. https://github.com/litmuschaos/chaos-charts/blob/master/charts/generic/container-kill/experiment.yaml#L62

**Comment by uditgaurav** [comment]
@ispeakc0de All other changes are synced added this one also.

**Comment by ksatchit** [comment]
Can the ansible-runner image tag be 1.4.1 instead of 1.4.0?

**Pull request merged**